kubernetes/mlflow-tracking-operator-master/Gopkg.toml

```diff
@@ -0,0 +1,75 @@
+# Force dep to vendor the code generators, which aren't imported just used at dev time.
+required = [
+  "k8s.io/code-generator/cmd/defaulter-gen",
```

Comment:
Are all thee go packages required to make `mlflow` work? If not--can remove extra ones?

Reply:
All these dependencies are required as we are utilizing the operator-sdk to setup the golang project to communicate with kubernetes. Do you think its possible to keep this config file? As it doesn't download anything unless you run `dep`

Reply:
As long as we need all these dependencies -- this config file is good. My question was about all these deps